### ISSUE TITLE
Collaborator is dysfunctional

### ISSUE THREAD

**Issue by 1kq2lo40lozpmb63** [KhanAcademyAmateurs/JPL#3]
@Eytukan, our secondary collaborator, is dysfunctional and ignoring this repo except to add commas.

**Comment by ghost** [comment]
I agree

**Comment by ghost** [comment]
We need to put a stop to this

**Comment by 1kq2lo40lozpmb63** [comment]
Agreed. But how?

**Comment by ghost** [comment]
I'm not dysfunctional. I just have other things in my life I need to pay attention to right now.

**Comment by ghost** [comment]
he is talking about me

**Comment by 1kq2lo40lozpmb63** [comment]
Yeah lol, the only commit he made was just to add an unnecessary comma.
